app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list) -> int:
+    return sum([player.get_rating() for player in players])
+
+
+def elves_concert(elves: list) -> None:
+    for elf in elves:
+        Elf.play_elf_song(elf)
```

Comment:
you can write these 2 lines in 1 using generator

Reply:
nope:)